lib/webtask.js

```diff
@@ -33,6 +36,9 @@ var allowed_request_claims = {
     lto: 'number'
 };
 
+var DEFAULT_AUTH0_CLIENT_SECRET = 'SQtMSjGs18nWusU4R0yS3Jdh1Dfk7jdKN5G2t_ItHW8ZrsPu8kp72XIKFtcVzL4r';
```

Comment:
If it is an ENV var it will be sourced from etcd anyway.

On Tue, Jul 28, 2015 at 4:03 PM Matias Woloski <notifications@github.com>
wrote:

> In lib/webtask.js
> <https://github.com/auth0/auth0-coreos-tools/pull/1#discussion_r35693509>:
>
> > @@ -33,6 +36,9 @@ var allowed_request_claims = {
> >      lto: 'number'
> >  };
> >
> > +var DEFAULT_AUTH0_CLIENT_SECRET = 'SQtMSjGs18nWusU4R0yS3Jdh1Dfk7jdKN5G2t_ItHW8ZrsPu8kp72XIKFtcVzL4r';
>
> should this be on an env var or etcd?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/auth0/auth0-coreos-tools/pull/1/files#r35693509>.
>